src/components/timeline/index.css

```diff
@@ -65,40 +65,57 @@
 .timelineSettingsHiddenTracks {
   /* Make sure and override all rules that have to do with button defaults. */
   position: relative;
-  height: 17px;
-  padding: 0 7px 0 20px;
+
+  /* 1px is allocated for the vertical separator line. */
+  width: calc(var(--thread-label-column-width) - 1px);
+  height: 20px;
+
+  /* 8px is intentional to align this text with the text above. */
+  padding: 0 20px 0 8px;
```

Comment:
We should start to use RTL-friendly logical properties. What about:
```css
padding-inline: 8px 20px;
```
?

Reply:
Isn't a piecewise conversion going to have a rather broken intermediate state, if you try RTL before the conversion is finished? I wonder if it would make more sense to do everything in one fell swoop.

Reply:
Yes, I thought the same thing as Markus actually. While writing this, I tried to support RTL here with the down triangle and other properties but I wasn't happy with the result in the end because it wasn't looking great on RTL due to surrounding elements not supporting it. If we change the padding to `padding-inline: 8px 20px;` right now, the rectangle on the left will overlap with the text.